### PR TITLE
[Index Card RPG Core 2e] "ICRPG Extended" character sheet created

### PR DESCRIPTION
**Pull request by Thomasvangrol**
Created a new character sheet for Index Card RPG Core 2e, with much more space and updated mechanics that have been added to the game in the last few years.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [x] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Thomasvangrol** [comment]
*The title has the sheets name: ICRPG extended
*The sheet adds new functionalities to the older sheets that were on roll20. In the last few years more rules were added to the game, which we included.
*It is a new sheet and we followed the standards of building character sheets.

**Comment by Anduh** [comment]
Does this use the same attributes as the old sheet?

**Comment by Thomasvangrol** [comment]
Only a few of the basic ones. We changed most of them and added a lot to cater for new rules.

**Comment by Anduh** [comment]
This sheet doesn't comply with minimum requirement, use of `<table>` isn't wanted. And that "stylesheet", have you made it work? I don't recall a single sheet have CSS though other methods than having it on the CSS file itself.
* https://wiki.roll20.net/Building_Character_Sheets#2._Good_Code

**Comment by Thomasvangrol** [comment]
Thank you for your comments. This was my first time using HTML and CSS and I guess I made some mistakes. The "stylesheet" was leftover from what I used as an example (left it as I though it might be important) and I'll try something else to not use the <table> tags.

File: ICRPG extended/ICRPGextended.html [hunk 1]
```diff
@@ -0,0 +1,594 @@
...
+<br />
+
+
+
+<div class='sheet-3colrow'>
+  <div class='sheet-col'>
+        <!-- Put the content for the first column here -->
+    <div id="attributes">
+      <h3 style="text-align: center; color: #FFF; background-color: #c00607;">Attributes</h3><br />
+                <table style="width: 50%;">
```
Comment (clevett) [inline ICRPG extended/ICRPGextended.html]: We no longer allow `<table>` to be used for basic layout per the Building Character Sheet guidelines.

**Comment by Thomasvangrol** [comment]
Alright, it took me a few days, but I have removed the table tag and a few issues with the layout so it should be complying with the rules of roll20.

File: ICRPG extended/ICRPGextended.html [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <input type='checkbox'>
+             <br />
+             Level Known <input type="number" name="attr_Fire" value="0">
+         </th>
+         <br/>
+         <br/>
+         <th>
```
Comment (clevett) [inline ICRPG extended/ICRPGextended.html]: `<th>` is tag that belongs in a table. They will need to be removed. What are they being used for here?

Comment (Thomasvangrol) [reply ICRPG extended/ICRPGextended.html]: > `<th>` is tag that belongs in a table. They will need to be removed. What are they being used for here?

Like I said before, I learned html especially for this project and I took a lot of tags from example sheets to learn it. Anyway, the tag is removed and was apparently not doing anything for the sheet. I hope the sheet is finally ready to upload now!

**Pull request merged**